### PR TITLE
Added CIPCode Csv upload

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
As a developer, I need to be able to upload the CIP codes into GIDS, so that data can be used to display new information as part of the CT redesign.
## Original issue(s)
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/19572

DB Migration: https://github.com/department-of-veterans-affairs/gibct-data-service/pull/788

## Testing done
Local testing 

## Screenshots
Screen Shot 2021-02-10 at 2 47 28 PM
Screen Shot 2021-02-10 at 2 50 37 PM

## Acceptance criteria
- [x] Db changes included

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]

**Comment by delli-gatti-michael** [comment]
This PR is for adding the ability to upload the data as a CSV only. Because of the format of the expected data, converting to other file types (.txt, .xls, .xlsx) will result in data loss

**Review by thilton-oddball — APPROVED** [review]
LGTM

**Review by thilton-oddball — CHANGES_REQUESTED** [review]
It looks like you will need to put just the migration and schema change in its own PR and remove them from this PR.

**Comment by ksantiagoBAH** [comment]
> It looks like you will need to put just the migration and schema change in its own PR and remove them from this PR.

I merged in the db changes and pulled them into this PR

File: spec/fixtures/cip_code.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CIPFamily,CIPCode,Action,TextChange,CIPTitle,CIPDefinition,CrossReferences,Examples
+01,01,This is a test of my code,no,"AGRICULTURE, AGRICULTURE OPERATIONS, AND RELATED SCIENCES.","Instructional programs that focus on agriculture and related sciences and that prepare individuals to apply specific knowledge, methods, and techniques to the management and performance of agricultural operations.",,
+01,01.00,No substantive changes,no,"Agriculture, General.",Instructional content is defined in code 01.0000.,,
```
Comment (ericboehs) [inline spec/fixtures/cip_code.csv:3]: Might want to fix the no new line at end of file. 🤷

**Review by ericboehs — APPROVED** [review]

**Review by thilton-oddball — APPROVED** [review]

**Pull request merged**